__tests__/components/ui/Badge.test.tsx

```diff
@@ -0,0 +1,41 @@
+import Badge from "@/components/ui/Badge";
+import "@testing-library/jest-dom";
+import { render, screen } from "@testing-library/react";
+import ReduxProvider from "../../../__mocks__/provider/ReduxProvider";
+
+jest.mock("next/router", () => ({
+    useRouter: () => ({
+      push: jest.fn(),
+      events: {
+        on: jest.fn(),
+        off: jest.fn(),
+        emit: jest.fn(),
+      },
+      isFallback: false,
+      pathname: "mocked-pathname"
+    }),
+  }));
+  
+
+describe("Badge", () => {
+    it("shoulld render the badge", () => {
+        render(
+        <ReduxProvider>
+        <Badge />
```

Comment:
Can't we create a custom render function which automatically adds the `ReduxProvider`. We don't need to duplicate everywhere we are doing a test

Reply:
That function can be something like 

```jsx
renderWithRedux(<Badge />)
```